APHKLogicExtractor/RC/Utils.cs

```diff
@@ -0,0 +1,20 @@
+using RandomizerCore.Logic;
+
+namespace APHKLogicExtractor.RC;
+
+public class Utils
+{
+    public static TermCollectionBuilder assembleTerms(Dictionary<string, List<string>> terms)
```

Comment:
capitalization, should be `AssembleTerms`

Reply:
Fixed